### ISSUE TITLE
v1.1.0 is missing ./lib/metric.js

### ISSUE THREAD

**Issue by jbryson3** [stefanpenner/async-disk-cache#41]
v1.1.0 is the latest version I can install from npm

```
Error: Cannot find module './lib/metric'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/jbryson3/code/arbiter2/node_modules/async-disk-cache/index.js:18:14)
```

**Comment by trentmwillis** [comment]
@jbryson3 we quickly published v1.1.1 which should fix this, sorry for the inconvenience!

**Comment by jbryson3** [comment]
Thanks, there must be a lag between when a version is published and when it appears on npm. I see it now.
